apps/backend/core/configuration/base.py

```
@@ -0,0 +1,355 @@
+    @field_validator("jwt_secret_key")
+    @classmethod
+    def validate_jwt_secret(cls, v: str) -> str:
+        """
+        Validate JWT secret key for security.
+
+        Checks:
+        - Minimum length (32 characters)
+        - Not a weak/common pattern
+        """
+        if len(v) < 32:
+            raise ValueError("JWT secret must be at least 32 characters")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)